spec.go

```diff
@@ -53,6 +57,8 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
+	// ExtraFiles specifies open file descriptors that should be passed to the process.
+	ExtraFiles []os.File `json:"extraFiles"`
```

Comment:
i think having runtime support for `LISTEN_FDS` and doing passthrough would be better than adding this runtime information in the spec that is to be serialized.